Formula/l/lando.rb

```diff
@@ -0,0 +1,33 @@
+class Lando < Formula
+  desc "Push button development environments"
+  homepage "https://docs.lando.dev"
+  url "https://github.com/lando/core/archive/refs/tags/v3.23.2.tar.gz"
+  sha256 "a3edf4c1d82ce6af725d9760122f74e8ed74d453140d12407bf5764f19b145d5"
+  license "GPL-3.0-or-later"
+
+  livecheck do
+    url :stable
+    regex(/^v?(\d+(?:\.\d+)+)$/i)
+  end
+
+  depends_on "node"
+
+  def install
+    system "npm", "install", "--production", *std_npm_args
+    system "npm", "add", "@yao-pkg/pkg@5.15.0"
+    system "npx", "pkg", "--config", "package.json", "--targets", "node20",
+     "--options", "dns-result-order=ipv4first", "bin/lando"
+    bin.install "dist/@lando/core" => "lando"
+  end
+
+  def caveats
+    <<~EOS
+      To complete the installation:
```

Comment:
Is this specific to homebrew? Otherwise it should not be a caveat

Reply:
Lando's default installation script does run `lando setup` as part of its execution, so it's not typically expected, but not strictly specific to homebrew. Mostly included it since the `lando-cli` recipe did as well.